packages/js/src/first-time-configuration/tailwind-components/steps/social-profiles/social-profiles-step.js

```diff
@@ -44,11 +44,12 @@ export default function SocialProfilesStep( { state, dispatch, setErrorFields }
 			"</a>"
 
 		),
-		{   // eslint-disable-next-line jsx-a11y/anchor-has-content
+		{	// eslint-disable-next-line jsx-a11y/anchor-has-content
 			a: <a
 				id="yoast-configuration-person-social-profiles-user-link"
 				href={ window.wpseoScriptData.userEditUrl.replace( "{user_id}", state.personId ) }
 				target="_blank" rel="noopener noreferrer"
+				data-hiive-event-name="clicked_update_or_add_profile | social profiles"
```

Comment:
I don't find it [here](https://docs.google.com/spreadsheets/d/1rGip-GJECM815lKY2lWDG07LEWOgDesLeQoh5o2E04w/edit#gid=0)?

Reply:
This is one of the events I have added, you can find all the events I have added in the [sheet](https://docs.google.com/spreadsheets/d/1Xwj--5P4VnVJigAZ1S7MrtkQqGwV6KQcL_KqyBPHItE/edit#gid=0) I linked in the `Context` part of this PR.